alphagov/paas-admin
# [#156168463] Remove the 'home' module

## What

This has been a starting point for the application to get moving around.
As the application grew, we no longer need it to exist nor do we want it
to be presented to our users.

We took the approach of redirecting the root URL to `/organisations`
endpoint, as this would be ideally a starting point for the user.

The redirection is and should be a temporary solution, for we need to
implement the router system that would fix the routes in general.

## How to review

- Code review
- Run tests (should be done by travis)
- Run the application using rafal's environment
  ```
  OAUTH_AUTHORIZATION_URL="https://login.rafalp.dev.cloudpipeline.digital/oauth/authorize" OAUTH_TOKEN_URL="https://uaa.rafalp.dev.cloudpipeline.digital/oauth/token" OAUTH_CLIENT_ID="rafal-login" OAUTH_CLIENT_SECRET="secret" API_URL="https://api.rafalp.dev.cloudpipeline.digital" UAA_URL="https://uaa.rafalp.dev.cloudpipeline.digital" NOTIFY_API_KEY="not-important-for-that-story" npm start
  ```
- See if by visiting the `http://localhost:3000/` you're redirected correctly to organisations page

:bee: Depending on #24 one or the other may need to be rebased. Both are changing `app.js`.
